server/task_executor_pitr_cutover.go

```diff
@@ -93,22 +100,23 @@ func (exec *PITRCutoverTaskExecutor) pitrCutover(ctx context.Context, task *api.
 	if err != nil {
 		return true, nil, err
 	}
-	defer conn.Close()
-
-	mysqlDriver, ok := driver.(*mysql.Driver)
-	if !ok {
-		log.Error("failed to cast driver to mysql.Driver")
-		return true, nil, fmt.Errorf("[internal] cast driver to mysql.Driver failed")
-	}
-	mysqlDriver.SetUpForPITR(exec.mysqlutil, binlogDir)
-
-	log.Debug("Swapping the original and PITR database.", zap.String("instance", task.Instance.Name), zap.String("originalDatabase", task.Database.Name))
-	pitrDatabaseName, pitrOldDatabaseName, err := mysql.SwapPITRDatabase(ctx, conn, task.Database.Name, issue.CreatedTs)
+	defer func() {
```

Comment:
Any reason we cannot do defer conn.Close()?

Reply:
> Any reason we cannot do defer conn.Close()?

https://github.com/bytebase/bytebase/pull/1705#discussion_r912605231